src/components/SectionContent.svelte

```diff
@@ -1,7 +1,6 @@
 <script lang="ts">
-  export let classes: string = '';
 </script>
 
-<section class={`${classes}`}>
+<section {...$$restProps}>
```

Comment:
Made this change in a few components. It's helpful because `class` will autodetect tailwind where as `classes` will not.